### DISCUSSION TITLE
Prime React style sheet is not loading in Browser after build

### DISCUSSION THREAD

**Discussion by Suma-ReactDev** [primefaces/community#2777] [PrimeReact]
I have used Vite React(V18) and Prime React Version 10.8.2. I have installed everything and cached the npm modules for offline Development then setup the SpringBoot to serve the frontend build from java/main/resources. It works fine on very few browsers and for other browsers it shows style sheet is loaded but I see 200 OK in console Networktab on browser. Can anyone please help me.....

**Comment by qwadrox** [comment]
@Suma-ReactDev Hi!
It sounds like you're facing a server configuration problem and your problem is not related to Prime React, but rather related to Spring Boot. You didn't specify which browser is working and what's not and didn't specify what happens when the style sheet is loaded but not working, so it's hard to recommend anything.

But you can try to open the developer tools in the browsers where the CSS isn’t working and look for any warnings or errors related to the styles. You might have CORS or MIME type errors. For example, you might have something like this: 
https://stackoverflow.com/questions/43266928/springboot-resource-interpreted-as-stylesheet-but-transferred-with-mime-type-t
